CuitService/CuitService.csproj

```diff
@@ -15,19 +17,4 @@
     <PackageReference Include="Tavis.UriTemplates" Version="1.1.1" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Update="public-keys\relay_prod.xml">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="public-keys\relay_test.xml">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="public-keys\doppler_test.xml">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
-    <None Update="public-keys\doppler_prod.xml">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
```

Comment:
Nice! and it is not necessary to copy `public-keys-dev/dev.xml` ❤️

Reply:
Dev public key is not being copied. Marking this as solved